libraries/AP_Airspeed/AP_Airspeed_Health.cpp

```diff
@@ -30,8 +31,9 @@ void AP_Airspeed::check_sensor_ahrs_wind_max_failures(uint8_t i)
 
     // update state[i].failures.health_probability via LowPassFilter
     float speed_accuracy;
-    if (AP::gps().speed_accuracy(speed_accuracy)) {
-        const float gnd_speed = AP::gps().ground_speed();
+    const AP_GPS &gps = AP::gps();
+    if (gps.speed_accuracy(speed_accuracy)) {
+        const float gnd_speed = gps.ground_speed();

```

Comment:
This change does not realy improve readability, code size or speed. Or am I missing something?

Reply:
The intention was to not request the reference twice. I'm happy to back that part out to keep it at a minimal change set, but the general path we've taken has been to reuse the references rather then query for them.